tests/ltp/patches/flistxattr03.patch

```diff
@@ -0,0 +1,64 @@
+diff --git a/testcases/kernel/syscalls/flistxattr/flistxattr03.c b/testcases/kernel/syscalls/flistxattr/flistxattr03.c
+index 49b463425..ab17f8703 100644
+--- a/testcases/kernel/syscalls/flistxattr/flistxattr03.c
++++ b/testcases/kernel/syscalls/flistxattr/flistxattr03.c
+@@ -13,12 +13,17 @@
+ * which can be used to estimate a suitable buffer.
+ */
+ 
++/* Currently xattr is not enabled while mounting root file system. Patch is
++ * to mount root file system with xattr enabled and then use it for the test.
++*/
++
++#include <stdio.h>
+ #include "config.h"
+ #include <errno.h>
+ #include <sys/types.h>
+ 
+ #ifdef HAVE_SYS_XATTR_H
+-# include <sys/xattr.h>
```

Comment:
not needed

Reply:
@letmaik I asked this to be fixed since #include looks more proper than # include. Don't you aggree? Do we really need to take it back?

Reply:
Why do we care about the formatting of external code? If you start here you have to be consistent and reformat them all, which doesn't make sense to me.

Reply:
The main reason this doesn't make sense is that we wouldn't have touched these lines in the patch if it wasn't for the formatting change. That's why I'm surprised.

Reply:
updated as per maik's review comment